### DISCUSSION TITLE
Can I have the current version of OSCP render in an iframe on WordPress?

### DISCUSSION THREAD

**Discussion by SherpaGreg** [bullhorn/career-portal#476] [Q&A]
I'm not a developer, just a marketing guy who knows enough code to really screw things up.  I'm wanting to craft a "careers" page that has a little more content than just the job listings.  Is it possible to have the OSCP render on an iFrame within the page, so I could still have some content and images before and after the frame? 

Does this make sense?

**Comment by pbearne** [comment]
@SherpaGreg 

There is a WordPress plugin that does this for you https://wordpress.org/plugins/bullhorn-oscp/

note: this breaks the indexing of the jobs by google 

So you might want to look at the solution from[ Matador Jobs](https://matadorjobs.com/)  this solves this and integrates fully in the WordPress

**Reply by SherpaGreg** [reply]
Thanks!  I think I read that the plugin doesn't work with the latest version of the OSCP, is that right?

**Reply by pbearne** [reply]
That I don't know. If so then even more reason to look elsewhere for solution that is supported :-)
